<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>351</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::West</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tabEngine">
    <attribute name="title">
     <string>Engine</string>
    </attribute>
    <widget class="QPushButton" name="newEngineButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>New</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteEngineButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>431</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>40</y>
       <width>461</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelEvelDir">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>EvalDir</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editEvalDir">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelBookDir">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>BookDir</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editBookDir">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>421</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelExtOptions">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Extended Options</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabOption">
    <attribute name="title">
     <string>Option</string>
    </attribute>
    <widget class="QTableWidget" name="tableEngineOptions">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>441</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelEngineOptions">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>13</y>
       <width>371</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Common engine options</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabPreference">
    <attribute name="title">
     <string>Preference</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Sound</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Piece Type</string>
     </property>
    </widget>
    <widget class="QToolButton" name="soundOnOffButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioPiece1">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>99</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Piece1</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioPiece2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>99</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Piece2</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>360</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
